src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,15 @@
+status = debug
```

Comment:
It would be nice if there were a note in the PR about this... Was there a specific need for this?  Was it for dev/testing purposes only and snuck into the PR by accident?

Reply:
Using the log4j2 has some advantages and ease of use over the default logger, like parameterized logging using {} instead of String.format etc. That said, I did not make use of log4j's capability throughout the Project, I modified them as and when I came across. This is just a starting point.